src/platform/packages/shared/kbn-unified-tabs/src/components/tabs_bar/tabs_bar.tsx

```diff
@@ -16,6 +16,7 @@ import type { TabItem, TabsServices, TabPreviewData } from '../../types';
 import { getTabIdAttribute } from '../../utils/get_tab_attributes';
 import { useResponsiveTabs } from '../../hooks/use_responsive_tabs';
 import { TabsBarWithBackground } from '../tabs_visual_glue_to_header/tabs_bar_with_background';
+import { TabsBarMenu } from '../tabs_bar_menu';
```

Comment:
Since the component implementation uses many inline props and function children we could wrap the whole `TabsBarMenu` into React.memo here so it updates only if props chnage.

Reply:
Perfect! Will do